packages/calcite-components/src/components/modal/modal.e2e.ts

```diff
@@ -28,6 +28,24 @@ describe("calcite-modal", () => {
     t9n("calcite-modal");
   });
 
+  describe("reflects", () => {
```

Comment:
Given `modal` is deprecated and not being updated in this PR, this is no longer required. I'll defer to you if you want to leave this extra test coverage.